src/scripts/main.js

```diff
@@ -354,7 +354,26 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+function createTableRow(tableId) {
+  people.age = people.born - people.died;
+  people.century = Math.ceil(people.died / 100);
```

Comment:
The calculation of 'age' and 'century' is incorrect. 'age' should be calculated as 'died - born', and 'century' should be calculated as 'Math.ceil(died / 100)'. Additionally, these calculations should be done for each person individually, not on the 'people' array.